myapi/utils.py

```diff
@@ -0,0 +1,37 @@
+def data_org(yelp_info, all_data_dicts, yelp_ids):
+    for data in yelp_info['businesses']:
+        data_dict = {}
+        data_dict['yelp_id'] = data['id']
+        data_dict['name'] = data['name']
+        data_dict['phone'] = data['phone']
+        data_dict['is_closed'] = data['is_closed']
+        data_dict['review_count'] = data['review_count']
+        data_dict['yelp_rating'] = data['rating']
+        data_dict['url'] = data['url']
+        data_dict['latitude'] = data['coordinates']['latitude']
+        data_dict['longitude'] = data['coordinates']['longitude']
+        data_dict['image_url'] = data['image_url']
+        a = ", "
+        data_dict['address'] = a.join(data['location']['display_address'])
+        data_dict['distance'] = data['distance']
+        all_data_dicts.append(data_dict)
```

Comment:
Im just a little confused about line 17. so we create data_dict, put all the restaurant info in there, but then append it to all_data_dicts, but in the end we are returning data_dict. Im not sure where we are using all_data_dicts with the appended data_dict...if that makes sense?

Reply:
Good question, I was returning data_dict because I thought that I had to return something to finish the method, but it actually turns out I don't have to.  I just removed that line and pushed it up